### DISCUSSION TITLE
How to create splash add (== "App Open ad" in admob wesite)

### DISCUSSION THREAD

**Discussion by ghost** [invertase/react-native-firebase#5048] [General]
<img width="266" alt="Screen Shot 2021-03-18 at 11 50 43 AM" src="https://user-images.githubusercontent.com/29586734/111570793-83e24400-87e0-11eb-9fd9-7cd4526b83b1.png">

In Admob website, you can create "App Open" ad unit, which is displayed in splash screen. However, I cannot find any documentation explaining how to create such splash ad using react-native-firebase.

Any thought?

**Comment by davidepalazzo** [comment]
Hi @easonccc1,

I don't believe there's an equivalent for that out of the box in RNF.

With that said, I think you can quiet easily create your own solution combining the [AppState](https://reactnative.dev/docs/appstate) with an [interstitial ad](https://rnfirebase.io/admob/displaying-ads#interstitial-ads), just an idea 😊

**Reply by Rebsos** [reply]
Don't do this, unless you want to get suspended or banned. "Do not place interstitial ads on app load" https://support.google.com/admob/answer/6201362#zippy=

**Reply by davidepalazzo** [reply]
That's not the case @Rebsos. @easonccc1 was looking to display an ad on "app_open" an event which happens when the user returns to the app. It isn't at launch/load or exit.

**Comment by ghost** [comment]
@davidepalazzo Thanks for the reply. Really appreciate that. I guess that's the only way out.

**Reply by davidepalazzo** [reply]
No worries!
